department-of-veterans-affairs/caseflow
# Frontend for assign hearings tab

Connects #6574

### Description
This PR contains the frontend code for the assign hearings tab on /hearings/schedule/assign. The backend updates will be in a separate PR.

### Testing Plan
1. Go to /hearings/schedule/assign
1. Select an RO
1. Confirm both tabs in the center of the page are displayed correctly.
